tests/src/test/java/com/oracle/weblogic/imagetool/tests/ITImagetool.java

```diff
@@ -965,7 +965,7 @@ void createMiiOl8slim(TestInfo testInfo) throws Exception {
             String tagName = build_tag + ":" + getMethodName(testInfo);
             String command = new CreateCommand()
                 .tag(tagName)
-                .fromImage(" oraclelinux", "8-slim")
+                .fromImage(" ghcr.io/oracle/oraclelinux", "8-slim")
```

Comment:
I think this leading space should be removed.

Reply:
Happily, I kept it because it was there before :)